packages/manager/modules/pci/src/projects/new/payment/credits/credits.html

```diff
@@ -41,6 +41,13 @@
              data-translate="pci_projects_new_credits_amount_min"
              data-translate-values="{ amount: $ctrl.creditMinPrice.text }">
         </div>
+        <p class="black-friday mt-2">
+            <span class="oui-status mr-1"
+                  data-translate="pci_project_new_payment_black_friday">
+            </span>
+            <strong data-translate="pci_project_new_payment_black_friday_info">
```

Comment:
Looks like this is displayed even if `$ctrl.isBlackFridayAsteriskVisible()` is `false` ?

Also, this is not a11y friendly, you could either:
- add a link to the end note (and a link to go back here 😄 )
- add the note as a `aria-label`
- have a link to the note as a `aria-describedby`

I think the best solution is the link, what do you think ? 😄

Reply:
i used aria-describedby